### ISSUE TITLE
Train/Test/Eval Split

### ISSUE THREAD

**Issue by Snixells** [tensorflow/models#9418]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [X] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [X] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [X] I checked to make sure that this issue has not already been filed.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/research/object_detection/

## 2. Describe the bug

I have a general question about how the Tensorflow 2 Object Detection API Works. Usually when training ML-Models the Data is split into train, test and validation parts. The Tensorflow Object Detection API only uses train and validation data. That's why my question is how the train/test split is done internally while training. Are always the same images used for training and testing or do they change at each step? How is all of that organized ? 

Thanks for your answer!

**Comment by Snixells** [comment]
@jvishnuvardhan Hey, are there any Ressources regarding the train/test/split of the object detection API ? Or is it possible that you describe it in a few sentences ?

**Comment by SamLee514** [comment]
If I understand your question correctly, I believe that there is no train/test split being run by the OD API during training. Like you said, the only image inputs the OD API takes are from training tfrecord files and validation tfrecord files, and if you configure the training pipeline properly, TensorFlow should utilize all the training images for training (the number per step depending on the batch size you set), and all the validation images for validation. If you wish to do evaluation on a third, held-out test set, that will have to be done after training.
